storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -2364,4 +2374,23 @@ public static long bitXor(Long a, Long b) {
         return rtn;
     }
 
+    /**
+     * converts a clojure PersistentMap to java HashMap
+     */
+    public static Map<String, Object> convertMap(Map map) {
```

Comment:
Will change method name. For parameter type, this method is called in `Executor.mkExecutor` which passes in `Map workerData`, which should later be replaced with a normal java map after worker is ported to java, so I personally prefer the generic Map type instead of using APersistentMap.